extensions/src/AWSSDK.Extensions.Logging.Log4NetAdaptor/AWSSDK.Extensions.Logging.Log4NetAdaptor.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>	  	  
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="log4net" Version="3.0.1" />
```

Comment:
`log4net` needs to be added to our NOTICE files (`Notice.txt` and `buildtools/NoticeForZips.txt`) as well.

Reply:
Updated and also added `Microsoft.Extensions.Logging.Abstractions` in the file.